content/docs/setup/kubernetes/minimal-install/index.md

```diff
@@ -69,7 +62,13 @@ to manage the lifecycle of Istio.
     $ helm init --service-account tiller
     {{< /text >}}
 
-1. Install Istio:
+1. Install the `istio-init` chart to bootstrap all the Istio's CRDs:
```

Comment:
This is the new first mention of CRDs in the document, please expand the term and link to the definition as it was before:
```suggestion
1. Install the `istio-init` chart to bootstrap all the Istio's [Custom Resource Definitions or CRDs for short](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions):
```

Reply:
done.